pkg/controller/worker/machines_test.go

```diff
@@ -66,7 +65,7 @@ var _ = Describe("Machines", func() {
 		chartApplier = mockkubernetes.NewMockChartApplier(ctrl)
 		statusWriter = mockclient.NewMockStatusWriter(ctrl)
 
-		// Let the client always the mocked status writer when Status() is called.
+		// Let the seedClient always the mocked status writer when Status() is called.
```

Comment:
```suggestion
		// Let the seed client always return the mocked status writer when Status() is called.
```
Similarly at other places.